configure.ac

```diff
@@ -1208,7 +1208,6 @@ if test x$enable_monotouch = xyes; then
 fi
 
 AC_ARG_ENABLE(cxx, [  --enable-cxx   compile some code as C++])
-AM_CONDITIONAL(ENABLE_CXX, test x$enable_cxx = xyes)
```

Comment:
I'd like to keep this one for consistency with the other `AC_ARG_ENABLE` + `AM_CONDITIONAL`.

Reply:
"Should" never be needed, but sure.